components/security-mgt/org.wso2.carbon.security.mgt.ui/src/main/resources/web/keystoremgt/view-keystore.jsp

```diff
@@ -169,7 +170,8 @@
 	                    <form>
 	                        <input value="<fmt:message key="import.cert"/>" type="button"
 	                               class="button"
-	                               onclick="location.href ='import-cert.jsp?keyStore=<%=keyStore%>'"/>
+	                               onclick="location.href
+                                           ='import-cert.jsp?keyStore=<%=Encode.forJavaScriptAttribute(keyStore)%>'"/>
```

Comment:
Here keyStore variable is being used in a query string. Use Encode.forUriComponent